tests/RedLockTest.php

```diff
@@ -146,17 +160,13 @@ public function testRefresh()
         });
 
         $redlock = new RedLock($this->servers);
-        $lock = $redlock->refreshLock([
-            'resource' => 'XYZ',
-            'validity' => 300000,
-            'token' => 1234,
-            'ttl' => 300000,
-        ]);
+        $lockObj = new Lock($redlock, 300000, 'XYZ', '1234', 300000);
+        $lock = $redlock->refreshLock($lockObj);
 
         $this->assertGoodRedisMake($caught_args);
-        $this->assertEquals('XYZ', $lock['resource']);
-        $this->assertTrue(is_numeric($lock['validity']));
-        $this->assertNotNull($lock['token']);
+        $this->assertEquals('XYZ', $lock->getResource());
+        $this->assertTrue(is_numeric($lock->getValidityTime()));
```

Comment:
There is an `assertIsNumeric` these tests could use instead

Reply:
Unfortunately it's not available with the version of the test methods that TestBench uses.